fauntlet/src/pen.rs

```diff
@@ -0,0 +1,130 @@
+use skrifa::raw::types::{Pen, PenCommand};
+
+/// Pen that eliminates differences between FreeType and Skrifa.
+///
+/// This covers three primary cases:
+///
+/// 1. All contours in FT are implicitly closed while Skrifa emits an
```

Comment:
Can we add tests for covering the behavior of the Regularizing Pen and these 3 examples? When I was working in Skia on this, I found this is an area in which it is easy to make mistakes - and that would affect our test outcomes.